### PR TITLE
Update screenshot and prop list for removed field.

### PR DESCRIPTION
**Pull request by stbof**
DOC-5766

### CODE REVIEW

**Review by xinhaoz — APPROVED** [review]
> Reviewed 3 of 6 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15275)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @xinhaoz)

File: v22.2/ui-transactions-page.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The **Statements** tab is selected. Click the **Transactions** tab and the **Act
 Active executions are polled every 10 seconds. Faster-running executions will potentially disappear upon each refresh.
 {{site.data.alerts.end}}
 
-The following screenshot shows the active statement execution for `SELECT city, id FROM vehicles WHERE city = 'paris'` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload):
+The following screenshot shows the active statement execution for `SELECT city, id FROM vehicles WHERE city = 'los angeles'` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload):
```
Comment (taroface) [inline v22.2/ui-transactions-page.md:60]: The screenshot shows an `UPSERT` statement involving `boston`. Or am I not understanding what's happening?

Comment (stbof) [reply v22.2/ui-transactions-page.md:60]: Thank you! I took so many screenshots and got them confused. I've committed the right ones now.

**Review by taroface — APPROVED** [review]
LGTM

**Pull request merged**